library/src/main/java/com/novoda/downloadmanager/lib/DownloadsRepository.java

```diff
@@ -55,7 +55,7 @@ public DownloadsRepository(SystemFacade systemFacade, ContentResolver contentRes
 
             return downloads;
         } finally {
-            downloadsCursor.close();
+            if (downloadsCursor != null) downloadsCursor.close();
```

Comment:
I'm not sure if this repo has different code style but we normally don't do one liner if statements.